giskard/llm/client/utils.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import os
+from enum import Enum
+from typing import Optional
+
+
+class ApiType(Enum):
+    openai = (0,)
+    azure = 1
```

Comment:
- constants should be uppercase
- enums start at 1 by default
- let’s find a more descriptive name
- acronyms in uppercase
see [PEP8](https://peps.python.org/pep-0008/).

```suggestion
class LLMClientAPI(Enum):  # maybe can find a better name
    OPEN_AI = 1
    AZURE = 2
```  

Also, this enum shold be moved in a more appropriate namespace than `utils`.